examples/Makefile

```diff
@@ -6,7 +6,7 @@ PATH_TO_MK = ../mk
 IMAGENAME = yanff-examples
 EXECUTABLES = demo dump Forwarding Firewall clonable_pcap_dumper kni
 
-all: nat tutorial
+all: nat tutorial antiDdos
```

Comment:
In `examples/antiDdos/Makefile`, `antiDDos` name is used:
```
+EXECUTABLES = generatorForAntiDDoS antiDDoS
```
Does this hurt consistency or it's intended and to be this way?

Reply:
Let's be consistent: all targets lower-case, no underscores.

Reply:
@Quasilyte , what did you mean? fix antiDdos to antiDDoS? 
@aregm, "Let's be consistent: all targets lower-case, no underscores." will be generatorforantiddos - it is not readable at all.

Reply:
@dkolistr, I was not aware about YANFF conventions. Intention was to notify about potential inconsistency (either antiDdos everywhere, or antiDDoS).
`generatorforantiddos` -> `ddosgen`? I can't help with naming, but "lower-case, no underscores" seems like enforcement for shorter names (like 7-8 chars, otherwise it's hard to read).
Or maybe "lower camel case" was meant (this would make sense, but will not help in `antiDdos` vs `antiDDoS`).

Reply:
@Quasilyte, Yes, I see about inconsistency, will fix. 
As for naming style - wait for Areg's comment